lib/Robo/Plugin/Commands/CodeCoverageCommands.php

```diff
@@ -50,18 +51,21 @@ class CodeCoverageCommands extends \Robo\Tasks
     use RoboTrait;
 
     /**
-     * Runs code coverage for travis ci
+     * Runs code coverage
+     * @param array $opts optional command line argument if using a continuous integration environment
```

Comment:
To be clear I meant something more like this:
```php
/**
 * Runs code coverage
 * @param array $opts
 * @option bool $ci Should be set to true if using a Continuous Integration environment.
 */
```

This is shown in the `Help` section of the Getting Started guide: https://robo.li/getting-started/